base/errorshow.jl

```diff
@@ -439,10 +439,8 @@ function show_method_candidates(io::IO, ex::MethodError, @nospecialize kwargs=()
                         end
                     end
                 end
-                if ex.world < min_world(method)
+                if ex.world < reinterpret(UInt, method.primary_world)
                     print(iob, " (method too new to be called from this world context.)")
-                elseif ex.world > max_world(method)
-                    print(iob, " (method deleted before this world age.)")
```

Comment:
I don't quite understand what changed to make this obsolete.

Reply:
Methods don't have world-ages bounds. They were never supposed to, and it doesn't make any logical sense to claim to have them.

Reply:
Then what does the `Method.max_world` field mean?

Reply:
Doesn't mean anything—that's why it's not correct to have and is now removed

Reply:
Does that mean that the `world` keyword argument to `hasmethod` should be deprecated (in 2.0)?

Reply:
No, while Methods don't have any particularly meaningful world age, the lookup algorithm for a method does